packages/core/bootstrap/src/lib/modules/overrider.ts

```diff
@@ -52,23 +52,24 @@ export class Overrider {
     const isOverridden: { [symbol: string]: boolean } = {}
     const alreadyWarned: { [symbol: string]: boolean } = {}
     for (const coinResponse of coinsResponse) {
-      const symbol = coinResponse.symbol
+      const symbolUpperCase = coinResponse.symbol.toUpperCase()
+      const symbolLowerCase = coinResponse.symbol.toLowerCase()
```

Comment:
Is it important to check uppercase and lowercase separately? At this point can we just make this completely case-agnostic?